.github/workflows/ci-sdk-example-integration-tests.yml

```
@@ -0,0 +1,32 @@
+  workflow_dispatch:
+
+jobs:
+  echo-example-integration-tests:
+    runs-on: arc-linux-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v6
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)